Makefile

```diff
@@ -22,9 +22,9 @@ $(SERVICES):
 
 install:
 	for file in $(BUILD_DIR)/*; do \
-		if [[ ! "$$file" =~ \.wasm$$ ]]; then \
-			cp "$$file" $(GOBIN)/propeller-`basename "$$file"`; \
-		fi \
+		if [ "$$file" = "$${file%%.wasm}" ]; then \
```

Comment:
Can we use Makefile `wildcard` directive here to contract the filtering and selection for copying?